src/App.tsx

```diff
@@ -1,26 +1,316 @@
-/* eslint-disable max-len */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import React, { useEffect, useState, useRef } from 'react';
+import { USER_ID } from './api/todos';
+import { Todo } from './types/Todo';
+import * as postService from './api/todos';
+import classNames from 'classnames';
+import { ToDoForm } from './components/ToDoForm';
 
 export const App: React.FC = () => {
+  // #region loadToDOs
+  const [todos, setToDos] = useState<Todo[]>([]);
+  const [loading, setLoading] = useState<boolean>(false);
+  const [loadingTodoId, setLoadingTodoId] = useState<number | null>(null);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [filter, setFilter] = useState<'active' | 'all' | 'completed'>('all');
+  const [tempToDo, setTempToDo] = useState<Todo | null>(null);
+
+  // Ref for the input element
+  const inputRef = useRef<HTMLInputElement>(null);
+
+  const loadToDos = () => {
+    setLoading(true);
+
+    postService
+      .getTodos(USER_ID)
+      .then(setToDos)
+      .catch(() => setErrorMessage('Unable to load todos'))
+      .finally(() => setLoading(false));
+  };
+
+  useEffect(() => {
+    loadToDos();
+
+    if (errorMessage) {
+      const timer = setTimeout(() => {
+        setErrorMessage(''); // Clear the error message after 3 seconds
+      }, 3000);
+
+      return () => clearTimeout(timer); // Cleanup the timer if the component unmounts or errorMessage changes
+    }
+  }, [USER_ID, errorMessage]);
```

Comment:
Fix it 
<img width="1189" alt="image" src="https://github.com/user-attachments/assets/de5f356f-f198-41d7-99ff-f754bf27a0de" />